mofacts/client/views/home/profile.js

```diff
@@ -72,7 +72,7 @@ Template.profile.events({
     selectTdf(
         target.data('tdfid'),
         target.data('lessonname'),
-        target.data('currentStimuliSetId'),
+        target.data('currentstimulisetid'),
```

Comment:
Were we not passing this correctly before? I assume it's either incorrect now or before with case mattering.

Reply:
It was not passing correctly before. `data-currentStimuliSetId` gets auto-lowercased when added to `target.data`.